### ISSUE TITLE
HIP Backend: OpenVX 1.3 - CMake Warning

### ISSUE THREAD

**Issue by kiritigowda** [GPUOpen-ProfessionalCompute-Libraries/MIVisionX#665]
```
clang-13: clang-13clang-13clang-13: : : clang-13clang-13: : clang-13: warning: argument unused during compilation: '-s' [-Wunused-command-line-argument]warning: 
argument unused during compilation: '-s' [-Wunused-command-line-argument]
warning: warning: warning: warning: argument unused during compilation: '-s' [-Wunused-command-line-argument]argument unused during compilation: '-s' [-Wunused-command-line-argument]warning: 
argument unused during compilation: '-s' [-Wunused-command-line-argument]argument unused during compilation: '-s' [-Wunused-command-line-argument]
argument unused during compilation: '-s' [-Wunused-command-line-argument]
```

**Comment by kiritigowda** [comment]
Also noticed that `openvx-hip` is generated

```
make
[ 11%] Built target openvx_hip
[ 68%] Built target openvx
[ 71%] Built target vxu
[100%] Built target runvx
```

**Comment by asalmanp** [comment]
> Also noticed that `openvx-hip` is generated
> 
> ```
> make
> [ 11%] Built target openvx_hip
> [ 68%] Built target openvx
> [ 71%] Built target vxu
> [100%] Built target runvx
> ```

the openvx-hip is built as an object library (please see [here](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/master/amd_openvx/openvx/hipvx/CMakeLists.txt#L39)). So this is correct.

**Comment by kiritigowda** [comment]
@asalmanp still seeing this warning on Ubuntu 20.04

```
[ 11%] Building CXX object amd_openvx/openvx/hipvx/CMakeFiles/openvx_hip.dir/vision_kernels.cpp.o
clang-13: clang-13: clang-13clang-13: clang-13clang-13: : : clang-13: warning: argument unused during compilation: '-s' [-Wunused-command-line-argument]
warning: warning: warning: warning: warning: warning: argument unused during compilation: '-s' [-Wunused-command-line-argument]argument unused during compilation: '-s' [-Wunused-command-line-argument]argument unused during compilation: '-s' [-Wunused-command-line-argument]argument unused during compilation: '-s' [-Wunused-command-line-argument]argument unused during compilation: '-s' [-Wunused-command-line-argument]argument unused during compilation: '-s' [-Wunused-command-line-argument]

```

**Comment by kiritigowda** [comment]
Issue fixed with #720

**Comment by kiritigowda** [comment]
Issue will be fixed when develop is merged into master
